src/app/plan/duck/actions/change_actions.ts

```diff
@@ -0,0 +1,104 @@
+
+
+interface IMigPlan {
```

Comment:
Exactly what I was hoping for with this PR, looking to bring some strong typing to the action interfaces. :tada: